components/Information.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+import { QUERIES } from "../utils";
+import type { Emp } from "../utils/umaApi";
+import { ethers } from "ethers";
+
+type Props = {
+  synth: Emp;
+};
+export const Information: React.FC<Props> = ({ synth }) => {
+  return (
+    <Wrapper>
+      <Heading>Token Information</Heading>
+      <Table>
+        <Row>
+          <div>Name:</div>
+          <div>{synth.tokenName}</div>
+        </Row>
+        <Row>
+          <div>Symbol:</div>
+          <div>{synth.tokenSymbol}</div>
+        </Row>
+        <Row>
+          <div>Address:</div>
+          <div>{synth.tokenCurrency}</div>
+        </Row>
+        <Row>
+          <div>Category:</div>
+          <div>{synth.category}</div>
+        </Row>
+        <Row>
+          <div>Expiry Date:</div>
+          <div>
+            {new Date(Number(synth.expirationTimestamp) * 1000).toUTCString()}
+          </div>
```

Comment:
I'd consider adding luxon (https://moment.github.io/luxon/#/) for handling Dates, it's something we should consider using as standard.

Reply:
Agree! I've added it!